apps/web/src/serlo-editor-integration/use-handle-learner-event.tsx

```diff
@@ -1,10 +1,98 @@
 import { LearnerEventData } from '@editor/plugin/helpers/editor-learner-event'
+import XAPI from '@xapi/xapi'
 
+import { useAuthentication } from '@/auth/use-authentication'
+import { useEntityData } from '@/contexts/uuids-context'
+
+const endpoint = 'https://watershedlrs.com/api/organizations/25975/lrs'
+const key = '97c9dcdcc2b2d8' //
+const secret = '44ac953d3f72a4'
```

Comment:
The empty comment makes me think that maybe, maybe, you wanted to add a TODO to hide `key` and `secret`?

Reply:
thanks, should be fine. removed comment